app/main.py

```diff
@@ -0,0 +1,109 @@
+from fastapi import FastAPI
+from fastapi.responses import JSONResponse
+
+from models.models import ParkingLot, Car, ParkingLotFullError, ParkingLotLocationEmptyError
+
+from config import (
+	TOTAL_SPOTS,
+	TIME_FRAME_TARIFFS
+)
+
+app = FastAPI()
+
+parking_lot = ParkingLot(TOTAL_SPOTS)
+
+@app.get("/")
+def index() -> JSONResponse:
+	""" Returns the index page """
+	return JSONResponse(content={"status": "ok"}, status_code=200)
+
+@app.get("/add")
+def add_car(car: str, tariff: str):
+	"""Add car to the parking lot endpoint handler
+
+	Returns:
+		JSONResponse:
+				error: JSON with the error message.
+				success: JSON with the ticket information.
+	"""
+	tariff = tariff.lower()
+	if tariff not in TIME_FRAME_TARIFFS:
+		return JSONResponse(
+			content={
+				"status": "error",
+				"description": f"Invalid tariff (Available options: {TIME_FRAME_TARIFFS})"
+			},
+			status_code=400
+		)
+
+	try:
+		ticket = parking_lot.add_car(Car(car), tariff)
+	except ParkingLotFullError as e:
+		return JSONResponse(
+			content={
+				"status": "error",
+				"description": str(e)
+			},
+			status_code=200
```

Comment:
Why status 200 if lot is full?

Reply:
Not add_car is ready to invalid tariff and full lot but can it fails for any other reason?